bumblefoot/src/bin/cmd/default.rs

```diff
@@ -39,9 +38,12 @@ pub fn command() -> Command<'static> {
         )
 }
 
-pub fn run(matches: &ArgMatches) -> AnyResult<bool> {
+pub fn run(matches: &ArgMatches) -> bumblefoot::CmdExit {
     log::info!("default cmd {:?}", matches.value_of("reporter"));
     println!("going to run {}", bumblefoot::CMD);
     bumblefoot::run();
-    Ok(true)
+    bumblefoot::CmdExit {
```

Comment:
Maybe we try returning `Result`? the reason I'm suggesting is because it would make it easier to write error producing code such as IO, network and more (e.g. opening files, etc.) and using `?` freely.

Reply:
yup, it sounds more flexible. 
the CMD commands return `Result<bumblefoot::CmdExit>`.
the exit logic is 
If `Ok,` take the CmdExit, print the text result if it exists, and exit with the given exit code
If `Err` log the error and exit with the default const configuration